### ISSUE TITLE
Separate files with the same keys raise a violation

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/dotenv-linter#20]
# Bug report

## What's wrong

I have two separate files:

```
KEY=1
```

and 

```
KEY=2
```

When I run `dotenv-linter file1 file2` it raise two violations:

```
file2:1 102 Found duplicate name: KEY
file2:1 102 Found duplicate name: KEY
```

## How is that should be

It should not raise any violations.
Since these two files are different. They can and should have the same keys.

**Comment by sobolevn** [comment]
Closed via https://github.com/wemake-services/dotenv-linter/commit/a7aa6fe9e7d4afe31ace4720ff5d636c0ffde6cc
